src/include/homeobject/common.hpp

```diff
@@ -11,7 +11,7 @@
 
 SISL_LOGGING_DECL(homeobject);
 
-#define HOMEOBJECT_LOG_MODS grpc_server, HOMESTORE_LOG_MODS, homeobject
+#define HOMEOBJECT_LOG_MODS grpc_server, HOMESTORE_LOG_MODS, homeobject, blobmgr
```

Comment:
Do we need another logging module; homeobject is not fine-grain enough?

Reply:
Since blob put and get have high number of requests, so kept a separate logging trace. Metadata related api's like pg and shard and other can be under homeobject.